index.html

```diff
@@ -5,8 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="css/styles.css" />
-    <title>Quiz-App</title>
+    <title>Quiz</title>
   </head>
 
-  <body></body>
+  <body>
+<header>
+    <div class="app">
+    <h1>Quiz App</h1>
+    <div class="quiz">
+        <h2 id="subhead">The quiz starts now!</h2>
+        </div>
+      </div>
+    </div>
```

Comment:
`<div>` is good, can also use `<section>` 🙂 
(but all is good semantics can be applied after main structure has been established 😄